src/main/java/com/farao_community/farao/gridcapa/task_manager/entities/ProcessFileStatus.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.gridcapa.task_manager.entities;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public enum ProcessFileStatus {
+    VALIDATED,
```

Comment:
Complete with other status expected (or remove the one not used of task) ?

Reply:
Removed the unsused for now. It is documented anyway.